github_helper.py

```diff
@@ -0,0 +1,137 @@
+"""
+github_helper provides several methods to help interface with Github
+"""
+
+import requests
+
+GITHUB_ORG = 'apache'
+SECRET_FILE = '../../github_auth/apache-beam.secret'
+SOURCE_REMOTE = 'github'
+TARGET_BRANCH = 'master'
+TARGET_REMOTE = 'apache'
+
+GITHUB_API_ROOT = 'https://api.github.com'
+GITHUB_REPO_FMT_URL = GITHUB_API_ROOT + '/repos/{0}/{1}'
+
+BOT_NAME = 'merge-bot'
+GITHUB_SECRET = '../../github_auth/mergebot.secret'
+
+
+class GithubHelper(object):
+    """
+    GithubHelper provides a lightweight authenticated wrapper for the Github
+    API.
+    """
+
+    def __init__(self, org, repo):
+        self.org = org
+        self.repo = repo
+        with open(GITHUB_SECRET, 'r') as key_file:
+            self.bot_key = key_file.read().strip()
+        print('Loaded key file.')
+
+    def fetch_prs(self):
+        """
+        Gets the PRs for the configured repository
+
+        :return: returns a JSON object containing the pull request if
+                         successful, else None.
+        """
+        p_url = pulls_url(self.org, self.repo)
+        print('Loading pull requests from Github at {}.'.format(p_url))
+        req = requests.get(p_url, auth=(BOT_NAME, self.bot_key))
+        if req.status_code != 200:
+            print('Oops, that didn\'t work. Error below,' +
+                        'waiting then trying again.')
+            print(req.text)
+            return None
+        print('Loaded.')
+        return req.json()
+
+    def fetch_comments(self, pr_num):
+        """
+        Gets the comments for a specified pull request
+
+        :return: returns a JSON object containing the comments if successful,
```

Comment:
```
"""
Gets the comments for the specified pull request.

Returns:
  A JSON object containing the comments if successful.
  None otherwsie.
"""
```

(May also be worth including a URL or short description of the JSON format, such as:

```
Returns:
  A JSON object containing the comments, formatted as:
  {'comments': [
    {'comment': ..., 'author': ...},
    {'comment': ..., 'author': ...},
    ...
    ]}
```